modules/central-configuration-options-operator.adoc

```diff
@@ -151,20 +151,101 @@ If no PVC with the given name exists, it will be created. The default value is `
+
+| `scannerV4.indexer.resources.limits`
+| Use this parameter to override the default resource limits for the Scanner V4 Indexer.
+
+| `scannerV4.indexer.resources.requests`
+| Use this parameter to override the default resource requests for the Scanner V4 Indexer.
+
+| `scannerV4.indexer.scaling.autoScaling`
+| When enabled, the number of Scanner V4 Indexer replicas is managed dynamically based on the load, within the limits specified.
+
+| `scannerV4.indexer.scaling.maxReplicas`
+| Specifies the maximum replicas to be used in the Scanner V4 Indexer autoscaling configuration.
+
+| `scannerV4.indexer.scaling.minReplicas`
+| Specifies the minimum replicas to be used in the Scanner V4 Indexer autoscaling configuration.
+
+| `scannerV4.indexer.scaling.replicas`
+| When autoscaling is disabled for the Scanner V4 Indexer, the number of replicas will always be configured to match this value.
+
+| `scannerV4.matcher.nodeSelector`
+| If you want this component to only run on specific nodes, you can configure a node selector by using this parameter.
+
+| `scannerV4.matcher.tolerations`
+| If the node selector selects tainted nodes, use this parameter to specify a taint toleration key, value, and effect for the Scanner V4 Matcher. This parameter is mainly used for infrastructure nodes.
+
+| `scannerV4.matcher.resources.limits`
+| Use this parameter to override the default resource limits for the Scanner V4 Matcher.
+
+| `scannerV4.matcher.resources.requests`
+| Use this parameter to override the default resource requests for the Scanner V4 Matcher.
+
+| `scannerV4.matcher.scaling.autoScaling`
+| When enabled, the number of Scanner V4 Matcher replicas is managed dynamically based on the load, within the limits specified.
+
+| `scannerV4.matcher.scaling.maxReplicas`
+| Specifies the maximum replicas to be used in the Scanner V4 Matcher autoscaling configuration.
+
+| `scannerV4.matcher.scaling.minReplicas`
+| Specifies the minimum replicas to be used in the Scanner V4 Matcher autoscaling configuration.
+
+| `scannerV4.matcher.scaling.replicas`
+| When autoscaling is disabled for the Scanner V4 Matcher, the number of replicas will always be configured to match this value.
+
+| `scannerV4.monitoring.exposeEndpoint`
+| Configures a monitoring endpoint for Scanner V4. The monitoring endpoint allows other services to collect metrics from Scanner V4, provided in a Prometheus- compatible format. Use `Enabled` to expose the monitoring endpoint. When you enable monitoring, {product-title-short} creates a new service, "monitoring", with port 9090, and a network policy allowing inbound connections to the port. By default, this is not enabled.
+
+| `scannerV4.scannerComponent`
+| Enables Scanner V4. The default value is `default`, which is not enabled. To enable Scanner V4, set this parameter to `enabled`.
```

Comment:
I don't know, is this case-sensitive or insensitive, @johannes94? I only remember having seen `Enabled, Default, Disabled`.

Also, maybe change the second sentence to something along the lines of

> The default value is ..., which has the meaning of `Disabled` for this release. In a future release the meaning of `Default` changes to `Enabled`.